src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax;
+
+public class Main {
+    private static final int NUMBER_OF_FIGURES = 6;
+
+    public static void main(String[] args) {
+        FigureSupplier figureSupplier = new FigureSupplier();
+        Figure[] figures = new Figure[NUMBER_OF_FIGURES];
+        for (int i = 0; i < figures.length / 2; i++) {
+            figures[i] = figureSupplier.getRandomFigure();
+        }
+        for (int i = figures.length / 2; i < NUMBER_OF_FIGURES; i++) {
+            figures[i] = figureSupplier.getDefaultFigure();
+        }
+        for (Figure figure : figures) {
+            figure.printFigureInfo();
+        }
```

Comment:
Do we actually need a separate loop? Could it affect the performance, if we had an array of billions of elements?

Let's print figures in the previous loops after their creation.